API/Configurations/OsuConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Configurations
+{
+    public class OsuConfiguration
+    {
+        public const string Position = "Osu";
+
+        [Required(ErrorMessage = "ApiKey is required!")]
+        public string ApiKey { get; set; } = string.Empty;
+
+        [Required(ErrorMessage = "ClientId is required!")]
+        [Range(1, long.MaxValue, ErrorMessage = "ClientId must be a positive, non-zero value!")]
+        public long ClientId { get; set; } = 0;
```

Comment:
I think default of 0 is kinda silly lol, is there any use to setting a default?

Reply:
possible null reference error grrrrrrrrrrrrr

Reply:
Ok I went back to look again and I guess it might have been Intellicode bugging because it's not giving me the warning anymore. I'll remove the unnecessary defaults